File: java/UKL/soal3.java
/*
Soal          : 3
Gambar        : A
Matrix kolom  : 6
*/
public class soal3 {   //gambar A
   public static void main(String[] args) {
      int a=5, 
          b=3,
          kolom=6,
          u=a,
          s=b; 
    System.out.println("Deret Aritmatikanya adalah"); 
    for(int i=0;i<kolom;i++){                            
        for(int j=1;j>0-i;j--){  
            System.out.print(u+"\t"); 
            u=u+b; 
            s=s+u;                     
        }  
        System.out.println(); 
    } 
    s=s-u; 
    System.out.println("Jumlah Deret Aritmatikanya adalah "+s);
    }
}

File: golang/UKL/soal3.go
/*
Soal          : 3
Gambar        : A
Matrix kolom  : 6
*/

package main

import "fmt"

func main() {
	a := 5
	b := 3
	kolom := 6
	u := a
	s := b
	fmt.Println("Deret Aritmatikanya adalah")
	for i := 0; i < kolom; i++ {
		for j := 0; j < i+1; j++ {
			fmt.Print(u, "\t")
			u = u + b
			s = s + u
		}
		fmt.Println()
	}
	s = s - u
	fmt.Println("Jumlah Deret Aritmatikanya adalah", s)
}
